tests/python/pants_test/util/test_dirutil.py

```diff
@@ -393,29 +393,29 @@ def test_rm_rf_no_such_file_not_an_error(self, file_name='./vanishing_file'):
       touch(file_name)
       rm_rf(file_name)
 
-  def assert_dump_and_read(self, test_content, dump_kwargs, read_kwargs):
+  def assert_write_and_read(self, test_content, write_kwargs, read_kwargs):
     with temporary_dir() as td:
       test_filename = os.path.join(td, 'test.out')
-      safe_file_dump(test_filename, test_content, **dump_kwargs)
+      safe_file_dump(test_filename, test_content, **write_kwargs)
```

Comment:
Do we need to update this to `safe_file_write`?

Reply:
> Update usages of safe_file_dump() across the repo, except in test_dirutil.py (which will be changed when the method is removed).

I was thinking we would wait until the deprecation period is over to do that, as until then, I think we should assume that we do "support" the use of the deprecated method (implying that we keep testing for it). I will add a TODO to remove `safe_file_dump()` testing when the deprecation period is over, I was just thinking that would be implicit.

Reply:
Done!

Reply:
This now needs to be reverted back to `assert_dump_and_read`. Sorry bout that!

Reply:
Done! Thanks for the catch!